sdk/resourcemanager/Azure.ResourceManager/src/ResourceListOperations.cs

```diff
@@ -128,11 +128,11 @@ private static GenericResourceContainer GetGenericResourceContainer(ResourceOper
             AsyncPageable<GenericResourceExpanded> result;
             if (scopeFilter == null)
             {
-                result = restClient.ListAsync(resourceFilters?.ToString(), expand, top, cancellationToken);
+                result = restClient.GetAllAsync(resourceFilters?.ToString(), expand, top, cancellationToken);
```

Comment:
In this class we have `ListAtContext` public method. Do we need to rename it as well?